### PR TITLE
build(bazel): remove tsec patch to enable runfiles on Windows

### PR DESCRIPTION
**Pull request by kormide**
@devversion @josephperrott @gregmagolan 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

We had to patch tsec to get it to use runfiles on Windows.

## What is the new behavior?

@uraj released a patch fix so we no longer need to patch it manually.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: fw-core, fw-animations

File: tools/defaults.bzl [hunk 1]
```diff
@@ -590,3 +591,10 @@ def api_golden_test_npm_package(**kwargs):
     _api_golden_test_npm_package(
         **kwargs
     )
+
+def tsec_test(**kwargs):
+    """Default values for tsec_test"""
+    _tsec_test(
+        use_runfiles_on_windows = True,
```
Comment (devversion) [inline tools/defaults.bzl:598]: Can you add a comment saying that we explicitly enabled runfiles for Windows in the `bazelrc`?

**Review by devversion — APPROVED** [review]
LGTM

Reviewed-for: global-dev-infra-approvers

File: tools/defaults.bzl (hunk 1 above)
Comment (kormide) [reply tools/defaults.bzl]: Done.

**Review by devversion — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit d2648d9d77d495d5c35b704cc2686d3c7497860d.

**Pull request closed without merging**